content/200-concepts/150-data-platform/01-about-platform/index.mdx

```diff
@@ -37,7 +37,7 @@ In order to add users, you need to use their GitHub username and they need to ha
 
 The Prisma Data Platform supports connecting to databases via static public IP addresses. This enables users to connect their databases to the platform while keeping their databases protected from the public internet.
 
-For more information, refer to our [documentation](/concepts/data-platform/static-ips) about using static IP addresses.
+For more information, refer to our [documentation](/concepts/data-platform/data-proxy/static-ips) about using static IP addresses.
```

Comment:
I thought the static IP feature was moving to be same hierarchy level as data proxy.